### PR TITLE
MWPW-108402 LANA logger script

### PR DESCRIPTION
**Pull request by chrischrischris**
Resolves: MWPW-108402

Test URLs:
- Before: https://main--milo--adobecom.hlx.page/
- After: https://<branch>--milo--adobecom.hlx.page/

### CODE REVIEW

File: libs/utils/lana.js [hunk 1]
```diff
@@ -1,20 +1,67 @@
...
- * LOG ALWAYS; NEVER ASSUME
- * @param {String} msg the message you want to log
- */
-export default function log(msg) {
-  // eslint-disable-next-line no-console
-  console.log(msg);
+var defaultOptions = {
+  clientId: '',
+  debug: false,
+  endpoint: '',
```
[the remark this answers is not in the document]
Comment (chrischrischris) [reply libs/utils/lana.js:4]: Right now they have to use `lana.setDefaultOptions` but I think if we are only going to have 2 options then we can hardcode those and have `lana.useProd()` or possibly set a global.  Thoughts?

[the remark this answers is not in the document]
Comment (chrischrischris) [reply libs/utils/lana.js]: Updated the PR with a `useProd` option that defaults to `true`

File: libs/utils/lana.js [hunk 2]
```diff
@@ -1,20 +1,86 @@
...
- */
-
-/**
- * LOG ALWAYS; NEVER ASSUME
- * @param {String} msg the message you want to log
- */
-export default function log(msg) {
-  // eslint-disable-next-line no-console
-  console.log(msg);
+var MSG_LIMIT = 5000;
```
Comment (jedjedjedM) [inline libs/utils/lana.js:1]: While akamai supports 8k URI strings, Adobe IO cuts out around 2200 characters.  Let's got with a 2k limit.

**Review by jedjedjedM — APPROVED** [review]

File: libs/utils/lana.js [hunk 3]
```diff
@@ -1,20 +1,86 @@
...
+  debug: false,
+  endpoint: 'https://www.adobe.com/lana/ll',
+  endpointStage: 'https://www.stage.adobe.com/lana/ll',
+  errorType: 'e',
+  sampleRate: 1,
+  implicitSampleRate: 1,
+  useProd: true,
+};
+
+function setClientdId(id) {
```
Comment (mokimo) [inline libs/utils/lana.js:14]: Was this supposed to be exposed with an extra d after client? (`setClientdId`)  @chrischrischris

Comment (chrischrischris) [reply libs/utils/lana.js:14]: Doh!  Fixed with https://github.com/adobecom/milo/pull/26

**Pull request merged**